code/backend/batch/AddURLEmbeddings.py

```diff
@@ -1,20 +1,21 @@
 import logging
 import traceback
 import azure.functions as func
-import sys
 
 from utilities.helpers.DocumentProcessorHelper import DocumentProcessor
 from utilities.helpers.ConfigHelper import ConfigHelper
```

Comment:
If I change this line, and all the other imports to be prepended with `backend.batch.` to:
```suggestion
from backend.batch.utilities.helpers.ConfigHelper import ConfigHelper
```

I don't need this in the test files:
```python
function_app_path = os.path.abspath(
    os.path.join(os.path.dirname(__file__), "../backend/batch")
)
sys.path.append(function_app_path)
```

Does that work for you?

Reply:
I was also going to suggest this, but then I realised that when this gets deployed, the `batch` folder becomes the root, so it doesn't know about `backend.batch` at runtime

Reply:
Gotcha, thanks!